platform/vs/docker-sonic-vs/supervisord.conf.j2

```diff
@@ -164,7 +164,7 @@ environment=ASAN_OPTIONS="log_path=/var/log/asan/teammgrd-asan.log{{ asan_extra_
 {% endif %}
 
 [program:zebra]
-command=/usr/lib/frr/zebra -A 127.0.0.1 -s 90000000 -M dplane_fpm_nl --asic-offload=notify_on_offload
+command=/usr/lib/frr/zebra -A 127.0.0.1 -s 90000000 -M dplane_fpm_sonic --asic-offload=notify_on_offload
```

Comment:
it is better if we do not change it by default, it is better to first introduce this as option and then wait till it becomes stable.

Reply:
Hi @lguohan

Many thanks for the review.

I added a compile-time option `ENABLE_DPLANE_FPM_SONIC`.

When this option is set, SONiC uses the new FPM module.
When this option is not set, SONiC uses the old FPM module and behaves exactly as it does today.

By default, this option is not set.

Reply:
i am fine that we build them into image by default, but do not enable it by default. if you can do it at runtime, it will be easier for you, and i am fine with that.

does this dplane_fpm_sonic support all features as dplane_fpm_nl?

Reply:
@lguohan

Yes, the `dplane_fpm_sonic` supports all features as `dplane_fpm_nl`. It is a copy of the old `dplane_fpm_nl`.

Indeed, it is easier to compile this by default and keep the old `dplane_fpm_nl` running by default.
I reverted the compile option and changed the supervisord config file to keep `dplane_fpm_nl` as default.